pandas/io/formats/format.py

```diff
@@ -32,6 +33,8 @@
     is_timedelta64_dtype,
 )
 from pandas.core.dtypes.generic import (
+    ABCCategorical,
```

Comment:
Not a blocker but generally have been moving away from using the ABC classes in annotations as they don't actually represent anything statically and just get replaced with `Any`.

Does importing the actual object impact things negatively here?

Reply:
removed ABCs